GPU/Common/DepalettizeShaderCommon.cpp

```diff
@@ -249,7 +249,8 @@ void GenerateDepalShaderFloat(ShaderWriter &writer, const DepalConfig &config) {
 		break;
 	case GE_FORMAT_5551:
 		if (config.textureFormat == GE_TFMT_CLUT8 && mask == 0xFF && shift == 0) {
-			sprintf(lookupMethod, "(index.a * 128.0 + index.b * 64.0 + index.g * 4.0)");  // we just skip A.
+			// Follow the intent here, and ignore g (and let's not round unnecessarily).
+			sprintf(lookupMethod, "floor(floor(index.a) * 128.0 + index.b * 64.0)");  // we just skip A.
```

Comment:
Is the comment just outdated ("skip A")?

-[Unknown]

Reply:
yes, removing